app/assets/javascripts/discourse/app/routes/tag-show.js

```diff
@@ -143,7 +150,7 @@ export default DiscourseRoute.extend({
   setupController(controller, model) {
     const noSubcategories = this.noSubcategories;
 
-    this.controllerFor("tag.show").setProperties({
+    this.controllerFor(this.controllerName).setProperties({
```

Comment:
What's the difference between `this.controllerFor(this.controllerName)` and the `controller` argument?